### DISCUSSION TITLE
Keda Http Add-on support streaming ?

### DISCUSSION THREAD

**Discussion by snirbenyosef** [kedacore/http-add-on#736] [Q&A]
Hey,
Thank you for this awesome addon!

i have a question:
before using the addon I used my pod (Creating virtual Service,etc) as Streaming response, i mean i send requests to the pod, and i get my answer as streaming.
after using the Addon and setting up the inspector of keda html addon, the streaming stopped and i'm getting the full response and not streaming in batches.

do Keda Http add-on support Streaming right now?

thanks.

**Comment by itayariel** [comment]
Hey I found that adding 
```
func (lrw *loggingResponseWriter) Write(bytes []byte) (int, error) {
	n, err := lrw.downstreamResponseWriter.Write(bytes)
        if f, ok := lrw.downstreamResponseWriter.(http.Flusher); ok {
		f.Flush()
	}
        ....
```
into loggingResponseWriter solves the issue
